### DISCUSSION TITLE
Overwrite get command Cypress (12.3.0)

### DISCUSSION THREAD

**Discussion by gugarzon** [cypress-io/cypress#25447] [Questions and Help] [answered]
Hi,

Could someone please help me, I just updated cypress to its latest version (12.3.0) and my commands were obsolete.

This is the error I have with (Cypress.Commands.overwrite)

(The following error originated from your test code, not from Cypress.
> Cannot overwite the get query. Queries cannot be overwritten.
When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.
Cypress could not associate this error to any specific test.
We dynamically generated a new test to display this failure.) 

Could you please tell me how to correct this?

Code:

Cypress.Commands.overwrite('get', (originalFn, selector, options) => {
   if ((typeof selector === 'string' || selector instanceof 'string') && selector.startsWith('*')) {
      const formattedSelector = selector.replace('*', '');
      return originalFn("[data-testid='" + formattedSelector + "']", options);
   } else {
      if (selector.includes('[') && !selector.includes("='") && selector.includes('.')) {
         selector = selector.replace('=', "='");
         selector = selector.replace(']', "']");
      }
      return originalFn(selector, options);
   }
});

I don't want to have to create another command, this would be an example of what I don't want to do

Cypress.Commands.addQuery('get2', (originalFn, selector, options)

**Comment by jjhelguero** [comment, marked as answer]
There is an [open issue](https://github.com/cypress-io/cypress/issues/25078) for overwriting a cypress command. Here is the latest [response](https://github.com/cypress-io/cypress/issues/25078#issuecomment-1371225546).
